From c85c139ffd7d2fe2d594a1d6479bee2d543731b2 Mon Sep 17 00:00:00 2001
From: gbhakteX <gurudattax.bhakte@intel.com>
Date: Fri, 27 Jul 2012 11:27:58 +0530
Subject: [PORT FROM MAIN] Latency optimization in bufferCount / MW (Audio android framework).

BZ: 26849

BufferCount for AudioSink/AudioTrack is changed to 2 from 4. This
is done to improve the latency for the audio buffer. This is also
needed to pass some of the CTS test cases.

Change-Id: I7c5845b9e97513c46c2d8886d31169f417a8844b
Orig-Change-Id: I3d1efed033e76f620e1d34afa32f4286a28f1f11
Signed-off-by: gbhakteX <gurudattax.bhakte@intel.com>
Reviewed-on: http://android.intel.com:8080/77920
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Reviewed-by: Kandasamy, Muthukumar <muthukumar.kandasamy@intel.com>
Reviewed-by: M, Arulselvan <arulselvan.m@intel.com>
Tested-by: M, Arulselvan <arulselvan.m@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/media/MediaPlayerInterface.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/media/MediaPlayerInterface.h b/include/media/MediaPlayerInterface.h
index 8fc72c3..61166fe 100644
--- a/include/media/MediaPlayerInterface.h
+++ b/include/media/MediaPlayerInterface.h
@@ -53,7 +53,7 @@ enum player_type {
 };
 
 
-#define DEFAULT_AUDIOSINK_BUFFERCOUNT 4
+#define DEFAULT_AUDIOSINK_BUFFERCOUNT 2
 #define DEFAULT_AUDIOSINK_BUFFERSIZE 1200
 #define DEFAULT_AUDIOSINK_SAMPLERATE 44100
 
-- 
1.7.4.1

